src/lib/oauth2-service.ts

```diff
@@ -345,7 +345,7 @@ export class OAuth2Service extends EventEmitter {
     // for the sake of security.
     //
     // This is *not* a real oAuth2 server. This is *not* to be run in production.
-    res.redirect(url.href); // lgtm[js/server-side-unvalidated-url-redirection]
```

Comment:
@poveden Dropping this as now alerts are expected to be managed through the GitHub website. Stangely, now that the comment has been removed, there's no longer any `open-redirect` alert (whereas we've got one `missing-rate-limiting` above.

Thoughts?

Reply:
Or could it be related to alert changes not being taken into account at pull-request time (https://lgtm.com/help/lgtm/automated-code-review-faqs#why-automated-code-review-reports-misses-alert)?

Reply:
It looks as a coincidence to me.

~Perhaps we should silence the rate limiting one (if it can be done?~ I can see that it cannot be suppressed in code. Shame!